org-caldav.el

```diff
@@ -158,6 +158,14 @@ doc-string of `org-agenda-skip-if'.  Any entry that matches will
 not be exported.  Note that the normal `org-agenda-skip-function'
 has no effect on the icalendar exporter.")
 
+(defvar org-caldav-days-in-past nil
+  "Number of days before today to skip in the exported calendar.
+This makes it very easy to keep the remote calendar clean.
+
+nil means include all entries (default)
+any number set will cut the dates older than N days in the past.
```

Comment:
```suggestion
any number set will cut the dates older than N days in the past."
  :type '(choice (const nil :tag "include all entries")
		 (integer :tag "number of days before today to skip in exported calendar")))
```

Not sure how this multiline suggestion work... that said, as most defvar has been made defcustom, this one should be a defcustom too.